internal/controller/controllers/agentserviceconfig_controller.go

```diff
@@ -825,6 +825,7 @@ func (r *AgentServiceConfigReconciler) newImageServiceDeployment(ctx context.Con
 			{Name: "ASSISTED_SERVICE_SCHEME", Value: "https"},
 			{Name: "ASSISTED_SERVICE_HOST", Value: serviceName + "." + r.Namespace + ".svc:" + servicePort.String()},
 			{Name: "REQUEST_AUTH_TYPE", Value: "param"},
+			{Name: "INSECURE_SKIP_VERIFY", Value: "false"},
```

Comment:
I think this needs to either be `true` or configurable in some way.

@filanov what do you think? What are the proportion of connected operator users who would object to us downloading the base isos insecurely?

Reply:
Good question, most of the use cases are disconnected environments that probably want to use secured connection. 
So first lets make it configurable, it can be true as default.

Reply:
Setting the value to `false` means to verify and use secured connection since it's inverse logic. 

For making it configurable, is it best to do that through the assisted service configmap and then have this environment variable read it through there?

Reply:
I don't think it belongs in the assisted service configmap as it's not configuration for the assisted service.

There are a few ways we could go about this.
1. Put it in the agent service config CRD
    - This is what I have done in the past as it is a property of the installation, but it doesn't seem to be something we're doing here too much
2. Make it an env var in the operator pod
    - We do this with various config currently specifically [here](https://github.com/openshift/assisted-service/blob/master/internal/controller/controllers/images.go)

I'm curious what @flaper87 and @mhrivnak think about these options.
Like I said, I'd generally prefer 1, but the convention in this operator seems to be 2 right now. Is that just to avoid too much churn in the agentserviceconfig API?

Reply:
`AgentServiceConfig` is definitely the right way to go. It's an API for configuring and deploying the assisted service and related containers.

Env vars on the operator should be limited to only those things that affect the behavior of the operator itself.